addons/packages/fluxcd-helm-controller/0.19.0/README.md

```diff
@@ -0,0 +1,259 @@
+    | Creating cluster role binding 'fluxcd-helm-controller-default-cluster-rolebinding'
+    | Creating package resource
+    / Waiting for 'PackageInstall' reconciliation for 'fluxcd-helm-controller'
+    \ 'PackageInstall' resource install status: Reconciling
+
+    Added installed package 'fluxcd-helm-controller'
+    ```
+
+5. Verify the package install by running:
+
+    ```shell
+    tanzu package installed get fluxcd-helm-controller
+    ```
+
+   For example:
+
+    ```shell
+    - Retrieving installation details for fluxcd-helm-controller...
+    NAME:                    fluxcd-helm-controller
+    PACKAGE-NAME:            helm-controller.fluxcd.community.tanzu.vmware.com
+    PACKAGE-VERSION:         0.19.0
+    STATUS:                  Reconcile succeeded
+    CONDITIONS:              [{ReconcileSucceeded True  }]
+    ```
+
+   Verify that `STATUS` is `Reconcile succeeded`
+
+    ```shell
+    kubectl get pods -n helm-system
+    ```
+
+   For example:
+
+    ```shell
+    $ kubectl get pods -n helm-system
+    NAME                               READY   STATUS    RESTARTS   AGE
+    helm-controller-7748954977-cgv59   1/1     Running   2          157m
+    ```
+
+   Verify that `STATUS` is `Running`
+
+## Try helm-controller
+
+1. Verify all the objects are installed:
+
+   This package would create a new namespace where all the elements of fluxcd will be hosted called `helm-system` you can verify the main components of `helm-controller` were installed by running:
+
+    ```shell
+    $ kubectl get all -n helm-system
+    NAME                                   READY   STATUS    RESTARTS   AGE
+    pod/helm-controller-7748954977-cgv59   1/1     Running   0          3m28s
+
+    NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/helm-controller   1/1     1            1           3m28s
+
+    NAME                                         DESIRED   CURRENT   READY   AGE
+    replicaset.apps/helm-controller-7748954977   1         1         1       3m28s
+    ```
+
+   you should get something really similar!
+
+2. Verify all the CRD were installed correctly:
+
+   The way you would communicate with `helm-controller` would be through its CRDs, this will be your main action point.
+
+   In order to check all the CRDs were installed you can run:
+
+    ```shell
+     $ kubectl get crds | grep ".fluxcd.io"
+     buckets.source.toolkit.fluxcd.io                                               2022-04-18T07:29:09Z
+     gitrepositories.source.toolkit.fluxcd.io                                       2022-04-18T07:29:09Z
+     helmcharts.source.toolkit.fluxcd.io                                            2022-04-18T07:29:09Z
+     helmreleases.helm.toolkit.fluxcd.io                                            2022-04-18T07:40:57Z
+     helmrepositories.source.toolkit.fluxcd.io                                      2022-04-18T07:29:09Z
+    ```
```

Comment:
A number of these CRDs are made available with the fluxcd-source-controller package.

In the example below, the `flux-system` namespace is used which is also created with the installation of the fluxcd-source-controller package.

Could you state that the fluxcd-source-controller needs to be installed as a prerequisite.

Reply:
i have put notes/comments that fluxcd-source-controller is a pre-requisite for these. I have also given steps to install the source controller as part of the Installation steps (1st step in Installation heading is to install pre-requisites)